src/bodyScrollLock.js

```diff
@@ -127,7 +127,7 @@ const setPositionFixed = () => window.requestAnimationFrame(() => {
       const bottomBarHeight = innerHeight - window.innerHeight;
       if (bottomBarHeight && scrollY >= innerHeight) {
         // Move the content further up so that the bottom bar doesn't hide it
-        document.body.style.top = -(scrollY + bottomBarHeight);
+        document.body.style.top = -(scrollY + bottomBarHeight) + 'px';
```

Comment:
Worth keeping it consistent to match the other areas (e.g. https://github.com/willmcpo/body-scroll-lock/pull/265/files#diff-643a4a126c8da9aa30956ce7bd9a85d41b9703c0b1ae59bf6c24b496afee10beR122-R123)

```suggestion
        document.body.style.top = `${-(scrollY + bottomBarHeight)}px`;
```